File: Medium-Questions/07-Reverse-Name/C/seven.c
#include <stdio.h>
#include <string.h>

int main(void)
{


    // reverse a string
    char name [40];
    printf("Enter a string: ");
    scanf("%s",name);

    printf ("After the reverse of a string: %s ", strrev(name));  
    return 0;

}   

File: Medium-Questions/07-Reverse-Name/C#/seven.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace seven
{
    class Program
    {

        public static string Reverse(string s)
        {
            // convert to array
            char[] charArray = s.ToCharArray();
            // Reverse Array
            Array.Reverse(charArray);
            // return result
            return new string(charArray);
        }

        static void Main(string[] args)
        {
            Console.Write("Enter Your Name: ");
            string name = Console.ReadLine();
            name = Reverse(name);

            Console.Write(name);

            Console.ReadKey();
        }
    }
}
